docs(developer): integration tests instruction (#4479)
carbon-design-system/carbon-for-ibm-dotcom: packages/web-components/docs/contributing-to-web-components.md

> gulp test:unit -s src/components/link-with-icon/__tests__/link-with-icon.test.ts -b Chrome -d -k
```

## Running build integration test

All commands below should be run at `packages/web-components` directory.

> You can run build integration test by:

Reviewer: ```suggestion
To run build integration tests:
```

```
> yarn test:integration:build
```

> You can run a specific set of UI test steps (e.g. running `tests/integration/build/masthead_steps.js` only) by:

Reviewer: ```suggestion
To run a specific set of UI test steps (e.g. running `tests/integration/build/masthead_steps.js` only):
```

```
> yarn test:integration:build masthead_steps
```

> By default Chrome runs in headless mode. You can show Chrome UI by:

Reviewer: ```suggestion
By default Chrome runs in headless mode. To enable Chrome UI while running tests:
```

```
> CI=false yarn test:integration:build
```

## Running UI integration test

All commands below should be run at `packages/web-components` directory.

> You can run UI integration test by:

Reviewer: ```suggestion
To run UI integration tests:
```

```
> yarn test:integration:ui
```

UI integration test runs against the Storybook environment. Running UI integration test will be faster by starting Storybook environment beforehand.

> You can run a specific set of UI test steps (e.g. running `src/components/lightbox-media-viewer/__tests__/lightbox-media-viewer_steps.js` only) by:

Reviewer: ```suggestion
To run a specific set of UI test steps (e.g. running `src/components/lightbox-media-viewer/__tests__/lightbox-media-viewer_steps.js` only):
```

```
> yarn test:integration:ui lightbox-media-viewer_steps
```

> By default Chrome runs in headless mode. You can show Chrome UI by:

Reviewer: ```suggestion
By default Chrome runs in headless mode. To enable Chrome UI while running tests:
```
